tools/bump-pack/src/turbo-diff.ts

```diff
@@ -38,6 +38,18 @@ import { setOutput } from "@actions/core";
 import { minimatch } from "minimatch";
 import { fileURLToPath } from "node:url";
 
+type TurboTaskOutput = {
+  package: string;
+  taskId: string;
+  directory: string;
+  cache: {
+    status: "HIT" | "MISS";
+  };
+  dependencies: string[];
+  inputs: { [key: string]: any };
+};
+type TaskChangeMap = { [key: string]: { cached: boolean; changes: boolean } };
```

Comment:
Can we include a comment maybe where this data structure is coming from? It looks as though the data is coming from a foreign API of some kind